rest/model/host.go

```diff
@@ -195,9 +195,19 @@ type APISpawnHostModify struct {
 }
 
 type APIHostScript struct {
-	Script string `json:"script"`
+	Hosts  []string `json:"hosts"`
+	Script string   `json:"script"`
 }
 
 type APIHostScriptResponse struct {
-	Output []string `json:"output"`
+	Output map[string]string `json:"output"`
```

Comment:
can we return a slice of script responses rather than a mapping?

Reply:
this has gone away